.github/workflows/helmfile_production_plan.yaml

```diff
@@ -0,0 +1,57 @@
+name: Production - Helmfile Diff
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "VERSION"
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  HELMFILE_FILE_PATH: ${{ github.workspace }}/helmfile
+  PRODUCTION_AWS_ACCOUNT: ${{ secrets.PRODUCTION_AWS_ACCOUNT }}
+
+jobs:
+  helmfile-diff:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Inject token authentication
+        run: |
+          git config --global url."https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/".insteadOf "https://github.com/"
```

Comment:
Oh interesting stuff going on here with the `insteadOf` instruction!

Reply:
This is a copy paste from our other GA's